docs/configuration.md

```diff
@@ -42,7 +72,7 @@ log:
 loglevel: debug # deprecated: use "log"
 storage:
 	filesystem:
-		rootdirectory: /tmp/registry
+		rootdirectory: /tmp/registry-dev
```

Comment:
Changing this at all was not on purpose (I assume my rebase failure).
Either way, I'm +1 on moving it - will also update the TLDR.